### PR TITLE
Advance Calculator Script in Python

### PR DESCRIPTION
**Pull request by amandp13**
#CH20 

### Advance Calculator Script in python

* Created a script to mimic a calculator using python using a regular expression module.
* It can do all the basic calculations continuously.
* No termination.

## Screenshots of I/O

advanceCalculator py - Visual Studio Code 8_21_2020 8_41_18 PM
advanceCalculator py - Visual Studio Code 8_21_2020 8_42_02 PM

### CODE REVIEW

File: Basic-Scripts/advanceCalculator.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#salutation users
+print("Welcome! I am your Calculator")
+print("Type 'quit' to terminate\n")
+
+#declaring global variables
+previous = 0
+run=True
+
+#defining non parametrized function
+def performMath():
```
Comment (SKAUL05) [inline Basic-Scripts/advanceCalculator.py]: What are u doing specifically in this function?

File: Basic-Scripts/advanceCalculator.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    global run 
+    equn = input("Give me something to calculate:")
+    if equn == 'quit':
+        print("You exit.")
+        print("See you next time.")
+        run= False
+    else:
+        print("You entered:",equn)
```
Comment (SKAUL05) [inline Basic-Scripts/advanceCalculator.py]: Can't see any calculations done!

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

File: Basic-Scripts/advanceCalculator.py (hunk 1 above)
Comment (amandp13) [reply Basic-Scripts/advanceCalculator.py]: this function is basically allowing users to calculate continuously....as in calculator one can do calculations with multiple variables and multiple operations.

File: Basic-Scripts/advanceCalculator.py (hunk 2 above)
Comment (amandp13) [reply Basic-Scripts/advanceCalculator.py]: Sorry as I committed the wrong basic file as I added the wrong file, now it's okay.
Please review.

**Comment by SKAUL05** [comment]
@amandp13 Can u specify Sample I/O in your code and document your function with PEP8 Guidelines.

**Comment by amandp13** [comment]
@SKAUL05 Done...please review

**Comment by amandp13** [comment]
@ankitdobhal Please review. #149

**Review by SKAUL05 — APPROVED** [review]
Good to go @ankitdobhal

**Comment by amandp13** [comment]
@ankitdobhal Please look into this.

**Pull request merged**